action.yml

```diff
@@ -3,20 +3,32 @@ description: 'This action installs reviewdog 🐶, runs phpcs 🐘 and pipes out
 inputs:
   reviewdog_token:
     description: 'The Github API token needed for Reviewdog'
+  path:
+    description: 'In which directory should phpcs be sniffing?'
+    required: false
+    default: '.'
+
 runs:
   using: 'composite'
   steps:
     - name: Install reviewdog
       run: $GITHUB_ACTION_PATH/install.sh
       shell: bash
       env:
-        REVIEWDOG_VERSION: 0.13.0
+        REVIEWDOG_VERSION: v0.13.0
         REVIEWDOG_TEMPDIR: ${{ runner.temp }}
     - name: Run phpcs and report to reviewdog
       run: |
         set +o pipefail
-        phpcs --config-set ignore_errors_on_exit 1
-        phpcs --config-set ignore_warnings_on_exit 1
+
+        # Change directory to where phpcs should do its thing.
```

Comment:
I think this comment is stating the obvious.

Reply:
Funny. Actually when I wrote the comment I thought:  @xendk is going to complain about this" - seriously! :)